app/index.html

```diff
@@ -74,5 +74,29 @@ <h4 class="modal-title">Javascript Code Editor</h4>
             </div><!-- /.modal-content -->
         </div><!-- /.modal-dialog -->
     </div><!-- /.modal -->
+
+    <div>    
+        <div class="modal" tabindex="-1" id="regionModal" role="dialog" data-backdrop="false" aria-labelledby="regionModal" aria-hidden="true">
+            <div class="modal-dialog modal-dialog-flat">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <strong>Select A Region</strong>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                    </div>
+                    <div class="modal-body" id="regionModalBody">
+                            <button type="button" class="regionButton" value="mypurecloud.com">US Eest</button>
```

Comment:
Typo in "Eest"

Also, include the domain in each label like `US East (mypurecloud.com)` and `EU West (euw2.pure.cloud)`. Users may not know the proper designation of their region, but should be familiar with the URL they use to log in to the Genesys Cloud app.

Reply:
Fixed